### PR TITLE
making the coffee images a bit bigger

### PR DESCRIPTION
**Pull request by billyfung**
As per #78, not sure what would be the optimal size

### CODE REVIEW

File: static/style.css [hunk 1]
```diff
@@ -53,3 +53,9 @@ footer.ui.footer {
 .size {
     font-size: 15px;
 }
+
+.ui.semismall.image {
+  width: 200px;
```
Comment (jinpark) [inline static/style.css:58]: did you check mobile?

File: static/style.css [hunk 2]
```diff
@@ -53,3 +53,9 @@ footer.ui.footer {
...
+  height: auto;
+  font-size: 0.92857143rem;
```
Comment (jinpark) [inline static/style.css]: also choose rem or px but stick with one

Comment (billyfung) [inline static/style.css]: I was going to ask you about that, why is the source done that way?

Comment (jinpark) [inline static/style.css]: its prob the js from semantic doing some calculations

File: static/style.css (hunk 1 above)
Comment (billyfung) [inline static/style.css:58]: what am I checking for? The site doesn't look so great on mobile

Comment (jinpark) [inline static/style.css:58]: since semismall is not part of semantic, will it do resizing correctly?

Comment (billyfung) [inline static/style.css:58]: ah, looks fine to me

**Comment by jinpark** [comment]
lgtm

**Pull request merged**